src/app/home/home.scss

```diff
@@ -1,4 +1,4 @@
-@import '../variables';
+@import '../variables.scss';

```

Comment:
Is this needed for the translation?  I heard there was difficulty compiling the sass, so could you post the error you're seeing here & we can decide if that requires a separate issue.

Reply:
I got an error 
$ ./node_modules/.bin/ng-xi18n
Error: Compilation failed. Resource file not found: D:/planet/src/app/variables